src/app/builders/components/BecomeBuilderBanner/BecomeBuilderBanner.tsx

```diff
@@ -5,7 +5,11 @@ import Image from 'next/image'
 import CollapsibleWithPreview from '@/components/CollapsibleWithPreview/CollapsibleWithPreview'
 import { FC } from 'react'
 
-const BecomeBuilderCollapsible: FC = () => {
+interface BecomeBuilderCollapsibleProps {
+  className?: string
+}
+
+const BecomeBuilderCollapsible: FC<BecomeBuilderCollapsibleProps> = ({ className }) => {
```

Comment:
Prefer the use of `CommonComponentProps`

personally, I'd write it as:

```suggestion
const BecomeBuilderCollapsible = ({ className }: CommonComponentProps): ReactElement => {
```